packages/api/src/modules/params/params.service.ts

```diff
@@ -46,15 +46,29 @@ export class ParamsService {
 
   private async initializeQuality() {
     const defaultQualities = [
-      { name: '4K', match: ['uhd', '4k', '2160', '2160p'], score: 4 },
-      { name: '1440p', match: ['1440', '1440p'], score: 3 },
-      { name: '1080p', match: ['1080', '1080p'], score: 2 },
-      { name: '720p', match: ['720', '720p'], score: 1 },
+      {
+        type: 'movie',
+        name: '4K',
+        match: ['uhd', '4k', '2160', '2160p'],
+        score: 4,
+      },
+      { type: 'movie', name: '1440p', match: ['1440', '1440p'], score: 3 },
+      { type: 'movie', name: '1080p', match: ['1080', '1080p'], score: 2 },
+      { type: 'movie', name: '720p', match: ['720', '720p'], score: 1 },
+      {
+        type: 'tvShow',
+        name: '4K',
+        match: ['uhd', '4k', '2160', '2160p'],
+        score: 4,
+      },
+      { type: 'tvShow', name: '1440p', match: ['1440', '1440p'], score: 3 },
+      { type: 'tvShow', name: '1080p', match: ['1080', '1080p'], score: 2 },
+      { type: 'tvShow', name: '720p', match: ['720', '720p'], score: 1 },
```

Comment:
Rename `tvShow` to `tvshow` it's easier to store and remember when this is in db

Reply:
@iam4x: I got what you have on your mind, but I highly recommended to use tvShow consistently as you did before, my reason is that I have created an enum for this in my other PR, and it can significantly decreases the possibility of making mistakes.

Reply:
Oh yes definitely :)